lib/MusicBrainz/Server/Data/ReleaseLabel.pm

```diff
@@ -41,7 +41,7 @@ sub _entity_class
 sub load
 {
     my ($self, @releases) = @_;
-    my %id_to_release = object_to_ids(@releases);
+    my %id_to_release = object_to_ids(grep { !$_->has_labels } @releases);
```

Comment:
I guess we don't need to worry about the labels having changed since they were loaded since this mostly only happens just after loading the release to begin with? Unless we load these on top of release data from edits or whatnot.